rust/src/plutus.rs

```diff
@@ -572,7 +572,7 @@ pub enum RedeemerTagKind {
 }
 
 #[wasm_bindgen]
```

Comment:
A lot of `plutus.rs` was code-generated originally but it's juts one line so it should be fine. `Copy` is a bit weird since it doesn't mean much from wasm (you till get memory freeing when passed by value like we got with `BigNum` iirc) and this class won't exist in rust (since we can use enums directly without needing to wrap them for wasm restrictions) after the regeneration.